mqt/qecc/cc_decoder/test_square_octagon_color_code.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+from square_octagon_color_code import SquareOctagonColorCode
+
+
+def test_d3():
+    d3_layout = SquareOctagonColorCode(3)
```

Comment:
small nit: the `d3_` is somewhat redundant here. just call this parameter `layout` in all the tests. Probably solved by adopting the pytest fixture.